manifests/c/coodesker/Coodesker/1.0.4.1/coodesker.Coodesker.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: coodesker.Coodesker
+PackageVersion: 1.0.4.1
+Installers:
+- Architecture: x64

```

Comment:
Please also add the x86 version - https://github.com/coodesker/coodesker-desktop/releases/download/v1.0.4.1/Coodesker-x86_1.0.4.1.exe